README.md

```diff
@@ -209,8 +209,15 @@ Duckling(jvm_started=False, parse_datetime=False, minimum_heap_size='128m', maxi
             is 2048m.
     """
 
-duckling.load():
-        """Loads the Duckling corpus"""
+duckling.load(self, languages=None):
```

Comment:
Please keep in mind if you change the API, that you update the README as well.
For the future, I plan to generate the docs via Sphinx, so this step won't be necessary anymore. 
Unfortunately, we still have to do it for now 🙁